### PR TITLE
Update archiver_slackr.rb

### PR DESCRIPTION
**Pull request by skoonin**
Fixed problem when archiving channels. It now correctly posts to the channel before archiving it, as well as skips previously archived channels and added more time into the Rate Limit waits.

### CODE REVIEW

**Review by newfurniturey — APPROVED** [review]
Looks good to me =]

File: archiver_slackr.rb [hunk 1]
```diff
@@ -134,16 +134,25 @@
...
-      channel: channel_id,
-      text: "Hello! This channel has been found ded and has been archived. Please contact IT with any memoriam or resurrection requests."
-      )
-    archiver_log.info { "[#{channel_name}] has been archived!" }
-    # add it to the historical log of archived_channels
-    File.open("slackr_archived_channels.log", "a") { |log| log.write("#{channel_name} was archived at #{run_date}\n") }
-    # sleep for slack rate limiting
-    sleep 1
+    is_archived = client.channels_info(channel: channel_id)
+    is_archived = is_archived.channel.is_archived
```
Comment (andrewjkerr) [inline archiver_slackr.rb]: We don't need to break this into two different assignments; we should just combine them for readability: :)

```ruby
is_archived = client.channels_info(channel: channel_id).channel.is_archived
```

File: archiver_slackr.rb [hunk 2]
```diff
@@ -134,16 +134,25 @@
...
+
+    if !is_archived  # if channel is not archived let's do stuff
```
Comment (andrewjkerr) [inline archiver_slackr.rb]: Can you please move the comment from beside the `if` to above the `if`? Like so:

```ruby
# If channel is not archived let's do stuff
if !is_archived
```

**Review by andrewjkerr — APPROVED** [review]
Looks good to me! I had two comments, but you're more than welcome to merge regardless. :)

**Pull request merged**